src/main/java/bit/javaoop/Main.java

```diff
@@ -1,4 +1,11 @@
 package bit.javaoop;
 
+import java.util.List;
+import java.util.ArrayList;
+
 public class Main {
+
+    public static void main(String[] args) {
+
```

Comment:
- [kolejność parametrów w metodach](https://softwareengineering.stackexchange.com/questions/101346/what-is-best-practice-on-ordering-parameters-in-a-function)
przy tworzeniu pracowników pierwszym parametrem konstruktora powinno być imię a salary drugie bardziej intuicyjnie tak jest:)
- Ceo powinien mieć metody supportujące funkcje dodawania/usuwania pojedyńczych pracowników
- Napewno testowałeś jakoś czy kod działa skoro nie zrobiłeś testów 😠 mogłeś zostawić co nieco w mainienie musiałbym tyle pisać xD
![image](https://user-images.githubusercontent.com/18233796/38467364-9d91d792-3b37-11e8-8fee-a360fa2f174c.png)